tests/foreman/ui/test_repository.py

```diff
@@ -334,6 +334,29 @@ def test_positive_delete(self):
                     self.repository.delete(repo_name)
 
     @run_only_on('sat')
+    @stubbed
+    @tier2
+    def test_negative_delete_puppet_repo_associated_with_cv(self):
+        """Delete a puppet repo associated with a content view - BZ#1271000
+
+        @Feature: Content Product - Delete repo
+
+        @Steps:
+        1. Create a new product
```

Comment:
Please add a blank line above to ensure better presentation for the generated HTML on the docs.